### PR TITLE
Auth for teams

### PR DESCRIPTION
**Pull request by captainmangoC4**
Verify findings submitted by teams
Allow wardens to submit as their team or as an individual

### CODE REVIEW

File: functions/submit-finding.js [hunk 1]
```diff
@@ -68,15 +70,17 @@ exports.handler = async (event) => {
 
   // ensure we have the data we need
   if (
-    !email ||
-    !handle ||
+    !emailAddresses ||
+    !emailAddresses.length > 0 ||
```
Comment (joshuashort) [inline functions/submit-finding.js:74]:
```suggestion
    !(emailAddresses.length > 0) ||
```

File: functions/submit-finding.js [hunk 2]
```diff
@@ -99,15 +102,31 @@ exports.handler = async (event) => {
...
+      username: user,
     });
     if (!confirmed) {
       return {
         statusCode: 401,
         body: "Unauthorized",
       };
     }
+    if (attributedTo !== user) {
+      const url = `${event.headers.origin}/.netlify/functions/getUser?id=${attributedTo}`;
```
Comment (joshuashort) [inline functions/submit-finding.js:114]: Does this work?

```suggestion
      const url = `/.netlify/functions/getUser?id=${attributedTo}`;
```

File: functions/submit-finding.js (hunk 1 above)
Comment (captainmangoC4) [reply functions/submit-finding.js:74]: good catch!

File: functions/submit-finding.js (hunk 2 above)
Comment (captainmangoC4) [reply functions/submit-finding.js:114]: Yes, it does. And in fact, node-fetch only allows using absolute urls. I can look into another work-around if this is not acceptable.

**Pull request merged**